processor/k8sattributesprocessor/doc.go

```diff
@@ -117,15 +140,19 @@
 // 1. Use the downward API to inject the node name as an environment variable.
 // Add the following snippet under the pod env section of the OpenTelemetry container.
 //
-//    env:
-//    - name: KUBE_NODE_NAME
-//      valueFrom:
-//  	  fieldRef:
-//  	    apiVersion: v1
-//  	    fieldPath: spec.nodeName
+//
+//    spec:
+//      containers:
+//      - env:
+//        - name: KUBE_NODE_NAME
+//          valueFrom:
+//            fieldRef:
+//              apiVersion: v1
+//              fieldPath: spec.nodeName
 //
 // This will inject a new environment variable to the OpenTelemetry container with the value as the
 // name of the node the pod was scheduled to run on.
+
```

Comment:
Is this line added accidentally?

Reply:
No It was already there, This is a part of Otel C container spec.nodeName which we are setting up as KUBE_NODE_NAME and using it in the k8sattributes filter to make sure we use fewer resources.
It was there in the previous version of the document as well.

Reply:
I mean the empty line added with this PR. Looks unrelated

Reply:
seems that was added accidentally , I have removed it